lib/src/transformer/impl_generation.dart

```diff
@@ -364,19 +364,20 @@ class ImplGenerator {
 
     typedMap.node.members
         .where((member) => member is FieldDeclaration)
-        .where((FieldDeclaration member) => !member.isStatic)
-        .forEach((FieldDeclaration field) {
+        .where((member) => !(member as FieldDeclaration).isStatic) // ignore: avoid_as
```

Comment:
Instead of this and the previous line you could just do:

```dart
typedMap.node.members
    .where((memeber) => member is FieldDeclaration && !member.isStatic)
    .forEach(...
```